package-lock.json

```diff
@@ -5158,6 +5158,10 @@
           "integrity": "sha512-tCfb2WLjqFAtXn4KEdxIhalnRtoKFN7nAwj0B3ZXCbQloV2tq5eDbcTmT68JJD3nRJq24/XgxtQKFIpQdtvmVw==",
           "dev": true,
           "requires": {
+            "commander": "^2.7.1",
+            "lodash.get": "^4.0.0",
+            "lodash.isequal": "^4.0.0",
+            "validator": "^8.0.0"
```

Comment:
This seems to be causing a build error. There's no `,` after this line, so `npm` fails ti parse the `package-lock`